### ISSUE TITLE
(French Tree 40s) Malagasy uprising happens despite independence given through referendum

### ISSUE THREAD

**Issue by Afrandez** [PlayerHOI/cwm_developer_version#123]
In addition; pre-war status of Madagascar returns at the end of civil war

**Comment by PlayerHOI** [comment]
Doesn't happen for me when I do the independence referendum...

**Comment by Afrandez** [comment]
Autonomy referendum hasn't changed uprising being triggered

**Comment by GreatIanEmpire** [comment]
Make sure that news event or whatever is removed, not sure if thats the problem, but it doesnt check for autonomy

**Comment by PlayerHOI** [comment]
Right, this was probably caused due to old events still being part of the French branch, now these are removed.
@Afrandez - Reopen this issue if you encounter this again after updating your local French branch.
